tests/lib/linter/linter.js

```diff
@@ -5470,4 +5472,102 @@ var a = "test2";
             assert.strictEqual(messages.length, 0);
         });
     });
+
+    // https://github.com/eslint/eslint/issues/14575
+    describe("directives in line comments", () => {
+        let messages;
+
+        it("//eslint-disable", () => {
+            const codes = linebreaks.map(linebreak => `//eslint-disable no-debugger${linebreak}debugger;`);
+
+            for (const code of codes) {
+                messages = linter.verify(code, { rules: { "no-debugger": 2 } });
+                assert.strictEqual(messages.length, 0);
+
+                messages = linter.verify(code, { rules: { "no-debugger": 0 } });
+                assert.strictEqual(messages.length, 0);
+            }
+        });
+
+        it("//eslint-enable", () => {
+            const codes = linebreaks.map(linebreak => `//eslint-disable no-debugger${linebreak}//eslint-enable no-debugger${linebreak}debugger;`);
+
+            for (const code of codes) {
+                messages = linter.verify(code, { rules: { "no-debugger": 2 } });
+                assert.strictEqual(messages.length, 1);
+            }
+        });
+
+        it("//eslint", () => {
```

Comment:
Can you add in some tests where there is a space after the `//`? I'd like to make sure we are covering all valid conditions.

Reply:
addressed in a2b3aaa46146f794a006b8f4014502d61ade9b85